fleet/store/store_test.go

```
@@ -0,0 +1,145 @@
+	require.NoError(t, err)
+	pend, err := s.PendingCommands(ctx, "ag_1")
+	require.NoError(t, err)
+	require.Len(t, pend, 1)
+	require.ErrorIs(t, s.AckCommand(ctx, cid, "ag_wrong", time.Now()), store.ErrNotFound, "ack scoped to the wrong agent must not apply")
+	require.NoError(t, s.AckCommand(ctx, cid, "ag_1", time.Now()))
+	pend, _ = s.PendingCommands(ctx, "ag_1")
+	require.Empty(t, pend)
+
+	v, err := s.Setting(ctx, "poll_interval")
+	require.NoError(t, err)
+	require.Equal(t, "", v)
```

golangci-lint (2.13.2)

[error] empty: use require.Empty

(testifylint)